### ISSUE TITLE
Alerter components may break the circuit execution

### ISSUE THREAD

**Issue by kwapik** [fluxninja/aperture#1022]
**Describe the bug**
Circuit stops execution after few minutes of running.

RCA:
1.  `Alerter` or `Load Actuator Alerter` are stuck on [`AddAlert`](https://github.com/fluxninja/aperture/blob/main/pkg/alerts/alerter.go#L29). 2. `AddAlert` is blocking because of noone actually reading from the channel and the channel buffer is full after 100 alerts occurences.
3. `AlertsReceiver` which is supposed to read from the channel is only reading it once, because of missing `for` loop [here](https://github.com/fluxninja/aperture/blob/main/pkg/otelcollector/alertsreceiver/receiver.go#L51).

**Steps to reproduce**
Just run the playground and wait.

**What version did you use?**
Checked this on latest main i.e. `2b4c6574`, but this bug existed before. See info below.

The faulty receiver was introduced in commit
```
commit 993ccc82048cb00c7870dfa2757c6e240a3114ad
Author: Krzysztof Kwapisiewicz <krzysztof.kwapisiewicz@fluxninja.com>
Date:   Mon Nov 7 13:24:38 2022 +0100

    Alerts pipelines (#893)
```

But is wan't possible for this bug to happen until the Alerter component was introduced. As we don't have an Alerter component in the example policies, the bug wasn't observed at the time.
```
commit 1342a07388503d5f9cc709a79bcfbe3b0b04b709
Author: Daria Bialobrzeska <88393714+DariaKunoichi@users.noreply.github.com>
Date:   Fri Nov 18 13:42:41 2022 +0100

    Alerter Component (#939)
```

The playground example is exposing the bug now, as we by default emit Alerts from the Load Actuator Alerter starting from commit
```
commit acaa129becfeadd3ccb5d8e197bcb3b0c9c50441
Author: Daria Bialobrzeska <88393714+DariaKunoichi@users.noreply.github.com>
Date:   Wed Dec 7 21:20:15 2022 +0100

    Load actuator alerter (#993)
```

In other words, this bug is present and can affect circuit in versions `v0.11.1`, `v0.12.0`, `v0.13.0`. Also, it will kill the playground circuit in version `v0.14.0-rc.1`.
